src/components/DeploymentForm.vue

```diff
@@ -61,6 +61,17 @@
       </p-content>
     </p-content>
 
+    <p-divider />
+
+    <p-content>
+      <h3 class="deployment-form__section-header">
+        Infrastucture overrides
```

Comment:
```suggestion
        Infrastructure overrides
```
would be good to move this to the localization module